lib/auth.js

```diff
@@ -52,6 +52,26 @@ var auth = {
     } else {
       return cb(null, false);
     }
+  },
+  /**
+   * @param  {object} req
+   * @param  {object} res
+   * @param  {object} cb
+   * New endpoint required as autentication ($fh.auth) requests not handled
+   * correctly in local development/Studio Preview -
+   * the js sdk dertermines envrironment mode from url string.
+   * New endpoint determines environment mode from env var,
+   * sends request (req) and calls user provided function (cb).
```

Comment:
normally in jsdoc, we put the description at the top, then parameter descriptions